lib-backcompat/chef/chef_fs/file_system/repository/file_system_root_dir.rb

```diff
@@ -24,6 +24,7 @@ module FileSystem
       module Repository
         class FileSystemRootDir < FileSystemEntry
           def initialize(file_path)
+            Chef.log_deprecation "This class is deprecated."
```

Comment:
Probably should include the class name so if you happen upon this in a log you have something to grep for.